frontend_tests/node_tests/templates.js

```diff
@@ -190,6 +190,19 @@ function render(template_name, args) {
     assert.equal(button.text(), "Subscribe");
 }());
 
+(function compose_all_everyone() {
+    var args = {
+        count: '101',
+        name: 'all'
+    };
+    var html = render('compose_all_everyone', args);
+    global.write_test_output("compose_all_everyone.handlebars", html);
+    var button = $(html).find("button:first");
+    assert.equal(button.text(), "YES");
+    var error_msg = $(html).find('span.compose-all-everyone-msg').text().trim();
+    assert.equal(error_msg, "Are you sure you want to message all 101 people in this stream?");
+}());
```

Comment:
yeah, that'd be pretty reasonable; if you can write a node test, they're way faster to run, but I'm not sure you can really write that test in Node.